ext/tags.go

```diff
@@ -22,6 +22,9 @@ var (
 
 	// PeerPort records port number of the peer
 	PeerPort = uint16Tag("peer.port")
+
+	// SamplingPriority determines the priority of sampling this Span.
+	SamplingPriority = uint32Tag("sampling.priority")
```

Comment:
Ok. Maybe worth switching back since there's also the change of accidental abuse here when overlooking the package-level methods. Alas, different PR.